### DISCUSSION TITLE
Deeplink to the Venmo app

### DISCUSSION THREAD

**Discussion by emilmicu** [braintree/braintree_ios#961] [General]
Hello,

I have a request to implement a deep-link to the Venmo app which opens and prepopulates a payment request. The URL would contain either the phone number or email address of the person I need to send the payment request to, then the current user decides wether to send or not the payment request. 

Is there a way to achieve this through the existing APIs? I haven't found any docs for this.

Thank you! 🙏

**Comment by scannillo** [comment]
Hi @emilmicu - thank you for using the Braintree SDK and offering feedback.

However, I am a bit confused as to what this request is? Can you elaborate further on what you mean and how it differs from our currently provided[ Venmo integration](https://developer.paypal.com/braintree/docs/guides/venmo/client-side)?

**Reply by emilmicu** [reply]
Hi @scannillo, sure. I already did the integration. What I'm confused about is actually making a payment request, which class would provide that? It's not very clear from the docs

**Reply by emilmicu** [reply]
To give more details, I need to make a payment request between two Venmo users. User1 would click a link in our app which opens the Venmo app with a payment request prepopulated with the amount. He sends the request, and User2 receives the request which he can accept or decline.

**Reply by scannillo** [reply]
The Braintree SDK handles Venmo payments between 1 user and a merchant application, not between two Venmo user accounts. Please consult our [developer docs](https://developer.paypal.com/braintree/docs) or reach out to [Support](https://developer.paypal.com/braintree/help) with questions specific to your integration.
